app.js

```diff
@@ -340,8 +340,39 @@ app.post('/newsale', function(request, response) {
   req.end();
 });
 
+app.get('/existinginstallation', function(request, response) {
```

Comment:
Could you change this path to 'getoneinstallation', please?
Here and in the ScheduleInstallation.jsx file.